src/router/task.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express')
+const Task = require('../models/task')
+const router = new express.Router()
+
+router.post('/', async (req, res) => {
+    const todo = new Task(req.body)
+    todo.save().then(() => {
+        res.send(todo)
+    }).catch((e) => {
+        res.send(e)
+    })
+})
+
+router.get('/', async (req, res) => {
+    Task.find({}).then((users) => {
+        res.send(users)
+        }).catch((e) => {
+        res.status(400).send(e)
+    })
+})
+
+router.get('/:id', async (req, res) => {
+    try {
+        const task = await Task.findOne({ _id: req.params.id })
+
+        if (!task) {
```

Comment:
Google if you express/mongoos can do that for you. If not can you decouple this?